docs/content/design/broker.md

```diff
@@ -59,7 +59,17 @@ Returns the dimensions of the datasource.
 
 Returns the metrics of the datasource.
 
+* `/druid/v2/datasources/{dataSourceName}/candidates/intervals/{comma-separated-intervals-in-ISO8601-format}/numCandidates/{numCandidates}`
+
+Returns segment information lists including server locations for the given datasource and intervals.
```

Comment:
if we elect to go with a short list of 5 server if `numCanditates` is absent, we should add this to the doc i guess.